tests/parser/functions/test_unsafe_math.py

```diff
@@ -43,12 +50,19 @@ def foo(x: {typ}, y: {typ}) -> {typ}:
         for (x, y) in itertools.product(xs, ys):
             expected = unsigned_to_signed(fn(x, y) % mod_bound, int_info.bits)
 
-            assert c.foo(x, y) == expected
+            assert c1.foo(x, y) == expected
+
+            c2 = get_contract(contract_2.format(typ, typ, op, y))
+            assert c2.foo(x) == expected
+
     else:
         # 0x80 has some weird properties, like
         # it's a fixed point of multiplication by 0xFF
         fixed_pt = 2 ** (int_info.bits - 1)
         xs += [0, 1, hi - 1, hi, fixed_pt]
         ys += [0, 1, hi - 1, hi, fixed_pt]
         for (x, y) in itertools.product(xs, ys):
-            assert c.foo(x, y) == fn(x, y) % mod_bound
+            assert c1.foo(x, y) == fn(x, y) % mod_bound
```

Comment:
nit: factor out the reused expression. `expected = fn(x, y) % mod_bound ... assert c1.foo(...) == expected`